proto/spire/api/agent/delegation/v1/delegation.proto

```diff
@@ -0,0 +1,75 @@
+syntax = "proto3";
+package spire.api.agent.delegation.v1;
+option go_package = "github.com/spiffe/spire-api-sdk/proto/spire/api/agent/delegation/v1;delegationv1";
+
+import "spire/api/types/selector.proto";
+import "spire/api/types/x509svid.proto";
+
+// The deletation service provides an interface to get the SVIDs of other
+// workloads on the host. This service is intended for use cases where a process
+// (different than the workload one) should access the workload's SVID to
+// perform actions on behalf of the workload. One example of is using a single
+// node instance of Envoy that upgrades TCP connections for different processes
+// running in such a node.
+//
+// The caller must be local and its identity must be listed in the allowed
+// clients on the spire-agent configuration.
+service Delegation {
```

Comment:
I am not sure the idiomatic naming here ... is it `Delegate` or `Delegation`? @azdagron do you have a preference?

Maybe `DelegateSVIDIssuance` or `DelegateIssuance` or `DelegateWorkloadIdentity`? I do think the term "delegate" is appropriate here

Reply:
Sticking with nouns is important, so I'd rather see "Delegated*" than "Delegate*". Even though this isn't _quite_ shaped like our other resource-oriented APIs, I wonder a resource-oriented name makes sense. Maybe `DelegatedIdentity`?

Reply:
Very good, we think `DelegatedIdentity` is more explicit, tks :D